webpack.config.js

```diff
@@ -22,6 +23,18 @@ const transpileNodeModules = [
   'tiptap-extensions',
 ].map(module => resolve('node_modules/' + module))
 
+const addonOutput = {
+  path: path.resolve(__dirname, "dist"),
+  publicPath: '',
```

Comment:
What is the reason for setting this to empty string? Isn't this the default?

Reply:
Sorry, this was a leftover and is not needed at all, dropped here [b4b01c4](https://github.com/demos-europe/demosplan-ui/pull/114/commits/b4b01c40910d04787bf59cc9f7d3a47df0ef1ecf)

Reply:
I was wrong here, the public path needs to be set to an empty string to make this config work. reverted here: https://github.com/demos-europe/demosplan-ui/pull/114/commits/7f4356a73c080a250cb91303069143da088d359a